lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -2145,12 +2078,14 @@ balanceTransactionSpec = describe "balanceTransaction" $ do
         (dummyTimeTranslationWithHorizon horizon)
         testStdGenSeed
 
-    utxo coins = UTxO $ Map.fromList $ zip ins outs
+    utxoWithBundles bundles = UTxO $ Map.fromList $ zip ins outs
```

Comment:
Slight restructuring of helper functions not needed anymore, but I think it's fine to keep.